libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -151,15 +151,14 @@
     width: 1%;
     height: 100%;
     animation: appPrompt-animate 5s linear forwards;
-    transform-origin: 0 0;
   }
 
   @keyframes appPrompt-animate {
     0% {
-        transform: scaleX(1);
+        width: 1%;
     }
     100% {
-        transform: scaleX(100);
+        width: 100%;
```

Comment:
I think we should investigate a bit more before falling back to animating `width`. The issue with it is that it can cause performance issues, transforms rely on GPU and are less of a performance bottleneck

Reply:
A quick reference on that - https://web.dev/articles/animations-overview

Reply:
Sure 👍 Will investigate a bit more

Reply:
Done. Found a solution that puts the GPU to work 👍